observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -574,6 +576,342 @@ config:
 |Returns errors up the pipeline and drops the payload. Implicit default.
 
 |===
+
+[id="tail-sampling-processor_{context}"]
+== Tail Sampling Processor
+
+The Tail Sampling Processor samples traces according to user-defined policies when all of the spans are completed. Tail-based sampling enables you to filter the traces of interest and reduce your data ingestion and storage costs.
+
+:FeatureName: The Tail Sampling Processor
+include::snippets/technology-preview.adoc[]
+
+This processor reassembles spans into new batches and strips spans of their original context.
+
+[TIP]
+====
+* In pipelines, place this processor downstream of any processors that rely on context: for example, after the Kubernetes Attributes Processor.
+* If scaling the Collector, ensure that one Collector instance receives all spans of the same trace so that this processor makes correct sampling decisions based on the specified sampling policies. You can achieve this by setting up two layers of Collectors: the first layer of Collectors with the Load Balancing Exporter, and the second layer of Collectors with the Tail Sampling Processor.
+====
+
+.Example of the OpenTelemetry Collector custom resource when using the Tail Sampling Processor
+[source,yaml]
+----
+# ...
+config:
+  processors:
+    tail_sampling: # <1>
+      decision_wait: 30s # <2>
+      num_traces: 50000 # <3>
+      expected_new_traces_per_sec: 10 # <4>
+      policies: # <5>
+        [
+          {
+            <definition_of_policy_1>
+          },
+          {
+            <definition_of_policy_2>
+          },
+          {
+            <definition_of_policy_3>
+          },
+        ]
+# ...
+----
+<1> Processor name.
+<2> Optional: Decision delay time, counted from the time of the first span, before the processor makes a sampling decision on each trace. Defaults to `30s`.
+<3> Optional: The number of traces kept in memory. Defaults to `50000`.
+<4> Optional: The expected number of new traces, which is helpful for allocating data structures. Defaults to `0`.
```

Comment:
```suggestion
<4> Optional: The expected number of new traces per second, which is helpful for allocating data structures. Defaults to `0`.
```

Also, I notice here that the stated default (0) doesn't match the YAML (10). Since the other two parameter explanations match their example values, I wanted to make sure there isn't a typo here.

Reply:
Thank you, Jeana. I checked the upstream docs. It is `0` indeed (hoping that the source is correct, hehe).

Reply:
Thank you, Jeana. I checked the upstream docs. It is `0` indeed.